mflowgen/cli.py

```diff
@@ -73,12 +73,13 @@ def parse_cmdline():
   p.add_argument( "-h", "--help",     action="store_true"          )
 
   # Run-related arguments
-  p.add_argument(       "--demo",     action="store_true"          )
-  p.add_argument(       "--design"                                 )
-  p.add_argument(       "--update",   action="store_true"          )
-  p.add_argument(       "--subgraph", action="store_true"          )
-  p.add_argument(       "--backend",  default="make",
-                                      choices=("make", "ninja")    )
+  p.add_argument(       "--demo",       action="store_true"          )
+  p.add_argument(       "--design"                                   )
+  p.add_argument(       "--update",     action="store_true"          )
+  p.add_argument(       "--subgraph",   action="store_true"          )
+  p.add_argument(       "--backend",    default="make",
+                                        choices=("make", "ninja")    )
+  p.add_argument(       "--graph_args"                               )
```

Comment:
Can we somehow merge the --subgraph and the --graph_args functionalities? Do they need to be two different options?

Also, I would prefer for the actual option to use dashes not underscores "--subgraph-args", and then python argparse will still convert this to underscores so you can reference it as "opts.subgraph_args".

Reply:
I think it makes sense to make them separate because there are many cases in which the graph_args don't actually need to be provided, but yes I will change the _ to -.

Reply:
The --subgraph option will be removed in a future PR as discussed